onnxruntime/python/tools/transformers/onnx_exporter.py

```diff
@@ -266,6 +266,11 @@ def load_pt_model(model_name, model_class, cache_dir):
     if hasattr(config, 'return_dict'):
         config.return_dict = False
 
+    num_layers = os.getenv('NUM_HIDDEN_LAYERS')
+    if num_layers is not None:
+        config.num_hidden_layers = int(num_layers)
```

Comment:
Some model like bart has also other attributes like encoder_layers, decoder_layers. Did you try it on encoder decoder model?

Reply:
Added encoder_layers and decoder_layers